threenums.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>   /*Armstrong Threenums it checks if all three numbers are equal, if they are not it gives you the largest number*/
+int main()
+{
+	printf("choose a number\n");
+	int x;
+	scanf("%d", &x);
+	printf("choose another number\n");
+	int y;
+	scanf("%d", &y);
+	printf("choose a final  number\n");
+	int z;
+	scanf("%d", &z);
+	if ((x==y)&&(x==z)&&(y==z)){
```

Comment:
you could have simplified this with the transitive property. if a == b and b == c, then a == b == c.